mathesar_ui/src/icons/index.ts

```diff
@@ -104,7 +104,7 @@ export const iconConstraintUnique: IconProps = { data: faSnowflake };
 export const iconDatabase: IconProps = { data: faDatabase };
 export const iconDbIdentifierDelimiter: IconProps = { data: faArrowRight };
 export const iconDisplayOptions: IconProps = { data: faPalette };
-export const iconExploration: IconProps = { data: faFileContract };
+export const iconExploration: IconProps = { data: faCrosshairs };
```

Comment:
@rajatvijay I mentioned in my original review that we should be using `faCompass` here to match Figma. This still seems to be unaddressed. Can you fix this?

Reply:
This does not have to be updated. I have added custom icons for tables and exploration from the figma design file in the branch I'm working in.